src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.h

```diff
@@ -4,6 +4,18 @@
 #include "ultra64.h"
 #include "global.h"
 
+typedef enum BgMizuMovebgType {
+    /* 0 */ MIZUMOVEBG_TYPE_FLOAT_PLATFORM_WEST,
+    /* 1 */ MIZUMOVEBG_TYPE_FLOAT_PLATFORM_OUTSIDE_CENTER_PILLAR,
+    /* 2 */ MIZUMOVEBG_TYPE_FLOAT_PLATFORM_INSIDE_CENTER_PILLAR,
+    /* 3 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_1,
+    /* 4 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_2,
+    /* 5 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_3,
+    /* 6 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_4,
```

Comment:
documenting what exactly 1,2,3,4 are is TODO right? (it's fine to not do it in this PR, just checking if that's the final name for you or not)

Reply:
1 is the boss room statue, 3 is the other ones and 2 and 4 are unused in MQ but does the same thing as 2

Reply:
maybe put that in a comment at least
eventually I guess they will be named by behavior difference if they're generic enough and not entirely hardcoded

Reply:
I added this:
```c
    /* 3 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_1, // Dragon Statue leading to the Water Temple boss room
    /* 4 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_2, // Unused on Master Quest, same behavior as ``MIZUMOVEBG_TYPE_DRAGON_STATUE_3``
    /* 5 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_3, // Dragon Statues used in Water Temple puzzles
    /* 6 */ MIZUMOVEBG_TYPE_DRAGON_STATUE_4, // Unused on Master Quest, same behavior as ``MIZUMOVEBG_TYPE_DRAGON_STATUE_3``